scripts-dev/make_full_schema.sh

```diff
@@ -147,7 +147,7 @@ python -m synapse.app.homeserver --generate-keys -c "$SQLITE_CONFIG"
 
 # Make sure the SQLite3 database is using the latest schema and has no pending background update.
 echo "Running db background jobs..."
-scripts-dev/update_database --database-config "$SQLITE_CONFIG"
+scripts/update_synapse_database --database-config --run-background-updates "$SQLITE_CONFIG"
```

Comment:
Typo here: `--run-background-updates` should come at the end. (Argparse thinks we've forgotten to specify a value for `--database-config`). Will fix as part of #13715.